README.md

```diff
@@ -1 +1,53 @@
-# AweBooks
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awe books
+
+> This is a  project I built to add and remove books using plain Javascript with objects
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo
+[Click Here](https://github.com/enoqueJonas/AweBooks)
+
+## Getting Started
+
+**To Create A Portfolio from this Repository feel free to contact me.**
+
+**To get a local copy up and running follow these simple steps.**
+- you can clone this repo by typing `git clone https://github.com/jellywiz/Awesome-Books.git`.
+- type `cd Awesome-Books.git` to access the project on terminal.
+
+
+## Authors
+👤 **Enoque Jonas**
+
+- GitHub: [@enoqueJonas](https://github.com/enoqueJonas)
+- Twitter: [@enoque_jonas](https://twitter.com/enoque_jonas)
+- LinkedIn: [@Enoque Macanda](https://www.linkedin.com/mwlite/in/enoque-macanda) 
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome.
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [OPTIONAL] I suggest you add the `LICENSE` file to your directory so then when we click on this link we can see your license and not get a `page not found` 😁 😉